test/functional/options/defaults_spec.lua

```diff
@@ -928,6 +932,9 @@ describe('stdpath()', function()
     -- Valid appnames:
     test_appname('a/b', 0)
     test_appname('a/b\\c', 0)
+    if not is_os('win') then
+      assert_log('Failed to start server: no such file or directory:', testlog)
```

Comment:
👍  NVIM_APPNAME does not affect server name on Windows.

Reply:
Uhh, isn't that the same thing? Maybe the intent is clearer you mean?

Reply:
yes, by "instead" I meant we could change the test code (not the behavior).

```
if is_os('win') then
  assert_nolog(...)
else
  assert_log('Failed to start server: no such file or directory:', testlog)
```